pkg/cloud/services/networking/securitygroups_rules.go

```diff
@@ -20,6 +20,11 @@ import (
 	infrav1 "sigs.k8s.io/cluster-api-provider-openstack/api/v1alpha6"
 )
 
+var (
+	CALICOCNI  = "calico"
+	DEFAULTCNI = CALICOCNI
```

Comment:
Not sure if the default should be defined here or better in the code within the switch case.
```suggestion
	CNICalico  = "calico"
```